crates/core/tedge_api/src/workflow/supervisor.rs

```diff
@@ -229,6 +298,138 @@ impl WorkflowSupervisor {
     }
 }
 
+/// The set of in-use workflow versions for an operation
+///
+/// - The current version is the version that will be used for a new command instance.
+/// - The current version might be none. This is the case when the command has been deprecated.
+/// - When a new command instance is initialized, the current version is stored as being in use.
+/// - When all the commands using a given version are finalized, these copies are removed (TODO).
```

Comment:
TODO: does ae56d97652 implement removal mentioned here? If so, `TODO` can be removed

Reply:
fixed: https://github.com/thin-edge/thin-edge.io/pull/3180/commits/3b7d30eb8d3a38f0efad50b2a650816078a66b0f